README.md

```diff
@@ -7,16 +6,25 @@ a powerful unified interface to use the same code to interact with ground-based
 satellite systems. It builds on top of popular open source libraries provided by
 [Adafruit](https://github.com/adafruit) and others.
 
-# Installing the SDK
-Installing the SDK is easy - it works like any other third party Arduino library. Just download 
-the SDK at https://s3-us-west-2.amazonaws.com/ardusatweb/ArdusatSDK.zip or clone this repository
-to your hard drive, then open the Arduino IDE, go to Sketch -> Import Library -> Add Library and
-navigate to your download (zip file or the directory cloned with git). You now should be able
-to use the SDK in your sketches.
+## Installing the SDK
+The first step to getting going is to install the Arduino Integrated Development Environment (IDE).
+Downloads for all major OS versions can be found at
+[http://arduino.cc/en/Main/Software](http://arduino.cc/en/Main/Software).
 
-# Using the SDK
+Once you have the IDE, installing the SDK is easy - it works like any other third party Arduino
+library. Just [download the SDK](https://s3-us-west-2.amazonaws.com/ardusatweb/ArdusatSDK.zip) or
```

Comment:
this link should be `https://www.ardusat.com/downloads/ArdusatSDK.zip` (caps important)